pkg/api/aim/runs.go

```diff
@@ -1039,3 +1039,8 @@ func toNumpy(values []float64) fiber.Map {
 		"blob":  buf.Bytes(),
 	}
 }
+
+// validateMetricNamePresent scans the query for metric.name condition
+func validateMetricNamePresent(query string) bool {
+	return strings.Contains(query, "metric.name == \"")
```

Comment:
This won't work once we have proper string comparison support implemented! See #13

Reply:
updated to catch more cases, see test